services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -33,11 +35,12 @@ data class Hanke(
         var modifiedAt: ZonedDateTime?,
 
         // Default for machine API's. UI should always give the save type.
+
         var saveType: SaveType? = SaveType.SUBMIT) {
 
-    var listOfOmistaja: List<HankeYhteystiedot> = arrayListOf()
-    var listOfArvioija: List<HankeYhteystiedot> = arrayListOf()
-    var listOfToteuttaja: List<HankeYhteystiedot> = arrayListOf()
+    var omistajat: MutableList<HankeYhteystieto> = arrayListOf()
```

Comment:
These lists can be initialized more simply:
```
var omistajat = mutableListOf<HankeYhteystieto>()
```

Reply:
Fixed all lists in Hanke